Kysect.CentumFramework.Sheets/Extensions/StringExtension.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+
+namespace Kysect.CentumFramework.Sheets.Extensions
+{
+    internal static class StringExtension
+    {
+        public static int ToColumnIndex(this string value)
+        {
+            if (string.IsNullOrEmpty(value))
+                throw new ArgumentException("Inappropriate column string");
```

Comment:
Сложное слово какое-то))0
```suggestion
                throw new ArgumentException("Invalid column index");
```

Reply:
done